generation/tasks/RunRivet.py

```diff
@@ -181,46 +177,35 @@ def run(self):
                 self.input()["analyses"]["collection"].targets.values()
             )
         )
-        for target in self.input()["analyses"]["collection"].targets.values():
-            with target.localize("r") as so_file:
-                so_file.copy_to_local(
-                    os.path.join(
-                        dirname, str(os.path.basename(so_file.path)).split("_", 1)[1]
-                    )
-                )
+        local_so_files = []
```

Comment:
Yeah, this can be definitely packed into a common util function.

Reply:
This is already an improvement over the older version. One can refactor this in a new PR. This is not in the scope for getting the plots done for the next publication.

Reply:
I will do a complete rewrite of the Merging Function in another PR (which inherently fixes this problem). This is out of scope for this PR.